src/beeware_docs_tools/build_pot_translations.py

```diff
@@ -0,0 +1,58 @@
+import subprocess
+from pathlib import Path
+
+SOURCE_DIR = Path.cwd()
+
+
+def generate_translated_pot_files(input_dir: Path, output_dir: Path) -> None:
+    subprocess.run(
+        [
+            "md2po",
+            f"--input={input_dir}",
+            f"--output={output_dir}",
+            "--pot",
+            "--duplicates=merge",
+        ],
+        check=True,
+    )
+
+
+def main():
+    docs_en_directory = Path(__file__).parent / "docs"
+    docs_en_directory.mkdir(parents=True, exist_ok=True)
+
+    try:
+        (Path(__file__).parent / "docs" / "en").symlink_to(
+            SOURCE_DIR / "docs" / "en", target_is_directory=True
+        )
+    except FileExistsError:
+        pass
+
+    output_directory = SOURCE_DIR / "docs" / "locales" / "templates"
+
+    output_directory.mkdir(parents=True, exist_ok=True)
+    generate_translated_pot_files(
+        input_dir=Path("docs") / "en",
+        output_dir=output_directory,
+    )
+
+    try:
+        (Path(__file__).parent / "docs" / "en" / "shared_content").symlink_to(
+            Path(__file__).parent / "shared_content", target_is_directory=True
+        )
+    except FileExistsError:
+        pass
+
+    generate_translated_pot_files(
+        input_dir=Path("docs") / "en" / "shared_content",
+        output_dir=Path(__file__).parent / "shared_content" / "locales" / "templates",
+    )
+
+    (Path(__file__).parent / "docs" / "en").unlink(missing_ok=True)
+    (SOURCE_DIR / "docs" / "en" / "shared_content").unlink(missing_ok=True)
+
+    (Path(__file__).parent / "docs").rmdir()
```

Comment:
I'm not following the cleanup strategy here; Is there a reason to not use the temporary directory approach that the build PO script uses? That avoids the need to do (potentially destructive) operations in the original source directory.

Reply:
This is an artefact of the shenanigans required to have the `msg*` string pair location paths be `docs/en/*`. If I use the temp directory, it sets the location path to the full temp directory path. The location path is literally what is provided as the "input" path. I fought with this for a long time when I first wrote this tooling, and there is no obvious way around it. It's not parsing anything for that location path, it's straight up making it whatever you provided to input.